Musigwa/SendIT-API
# #162110226 change order status by the given id

## What does this PR do (?)

It updates the specific order's status

## Description of Task to be completed (?)

The following Route should be working to enable the admin to change the status of the given parcel_Id

- PUT /parcels/<parcel_id>/status

## How should this be manually tested (?)

After cloning this repo and switch to this branch, run the dependencies installation using the Node package manager of your choice,

- Using any RESTful Client tool (Usually the postman), provide the parcel_id you want to update within the request URL, set the new status within the request body,
- after success, you should get the response containing your target parcel information with the updated status.

Key: Content-Type value: application/json

## Any background context you want to provide (?)

I'm using ES6 features in this branch, please be aware of the transpiler (.babelrc file) configurations.

## What are the relevant pivotal tracker stories (?)

1. 162110226
